test/plugins/features/movies/controller.test.js

```diff
@@ -82,15 +82,49 @@ describe('movie controller', () => {
 
   describe('retrieve movies with locations', () => {
 
+    beforeEach(async () => {
+      await Knex('locations_movies').insert(testLocMov);
+    });
+
     it('retrieves a movie with the related locations', async () => {
       const query = {};
       const movies = await Controller.list(query);
 
-      expect(movies.length).to.eql(2);
+      expect(movies.length).to.eql(3);
       expect(movies.models[0].get('name')).to.eql(testMovie1.name);
       expect(movies.models[0].relations.locations.models[0].attributes.name).to.eql(testLocation.name);
     });
 
   });
 
+  describe('add location to movie', () => {
+
+    it('adds a location to a movie', async () => {
+      const payload = { locationName: testLocation.name };
+      const movie = await Controller.addLocationToMovie(testMovie2.id, payload);
+
+      expect(movie.relations.locations.length).to.eql(1);
+      expect(movie.relations.locations.models[0].get('name')).to.eql(payload.locationName);
+    });
+
+    it('attempts to add a location to a movie twice', async () => {
```

Comment:
Same comment here about expected behavior: possibly `silently fails when adding...`. Or `does not add a location to a movie twice`, something like that.